1.9/administration/installing/custom/remove-a-node.md

```diff
@@ -0,0 +1,66 @@
+---
+post_title: Removing Agent Nodes
+nav_title: Remove Node
+menu_order: 801
+---
+
+You can remove agent nodes from an active DC/OS cluster. 
+
+When you change Mesos attributes (`⁠⁠⁠⁠/var/lib/dcos/mesos-slave-common`⁠⁠⁠⁠) or resources (⁠⁠⁠⁠`/var/lib/dcos/mesos-resources`⁠⁠⁠⁠), you have to remove the agent node and re-register it with the master under a new UUID. The master will then recognize the new attributes and resources specification.
+
+### Prerequisites:
+
+*   DC/OS is installed using the [custom](/docs/1.9/administration/installing/custom/) installation method.
+*   SSH installed and configured. This is required for accessing nodes in the DC/OS cluster.
+
+
+# Define a maintenance schedule
+You can define a maintenance schedule to evacuate your tasks prior to re-registering the UUID. ⁠⁠⁠All tasks that are running on the agent will be killed when you re-register the UUID. Mesos treats re-registration as a new agent.
+
+1.  Define a maintenance schedule and then invoke the `⁠⁠⁠⁠machine/down` endpoint with a JSON definition that includes the agents. For example, [here](https://github.com/vishnu2kmohan/dcos-toolbox/blob/master/mesos/maintain-agents.sh) is a maintenance schedule and [here](https://github.com/vishnu2kmohan/dcos-toolbox/blob/master/mesos/down-agents.sh) is down agents. 
```

Comment:
You should also include an example of the Maintenance schedule JSON:
https://github.com/vishnu2kmohan/dcos-toolbox/blob/master/mesos/agent-maintenance-schedule-example.json